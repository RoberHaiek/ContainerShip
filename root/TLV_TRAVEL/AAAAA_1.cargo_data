#<uid>,<weight>,<dstPort>
#container1,1,AAAAA
#container2,1,BBBBB
#container3,1,CCCCC
#container4,1,DDDDD
CSQU0000001,1,BBBBB
CSQU0000002,1,BBBBB
CSQU0000003,1,BBBBB
CSQU0000013,1,BBBBB