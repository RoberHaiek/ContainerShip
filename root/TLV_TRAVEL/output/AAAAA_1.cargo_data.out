CSQU0000001,R,-1,-1,-1
CSQU0000002,R,-1,-1,-1
CSQU0000003,R,-1,-1,-1