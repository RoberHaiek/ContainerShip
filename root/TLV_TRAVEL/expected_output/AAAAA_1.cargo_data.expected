CSQU0000001,L,1,1,1
CSQU0000002,L,1,1,1
CSQU0000003,L,1,1,1
#CSQU0000004,L,1,1,1